.github/workflows/docsite-build-deploy.yml

```diff
@@ -31,33 +31,18 @@ jobs:
         run: |
           sudo apt-get install -y pandoc
 
-      - uses: r-lib/actions/setup-r@v2
-        with:
-          extra-repositories: https://tiledb-inc.r-universe.dev
-
-      - uses: r-lib/actions/setup-r-dependencies@v2
-        with:
-          working-directory: ./api/r/cellxgene.census
-          extra-packages: any::rcmdcheck, any::styler, any::roxygen2, any::pkgdown
-          needs: check
-          cache: true
-
       - name: Build Sphinx website
         run: |
           cd docs/
           make clean && make html
           cp -r _build/html/* ../docsite/.
 
-      - name: Build pkgdown website
-        run: |
-          cd api/r/cellxgene.census
-          Rscript -e 'pkgdown::build_site()'
-          cd ../../../
+          # Copy R pkgdown docs (static HTML checked into git)
           mkdir -p docsite/r
           cp -r api/r/cellxgene.census/docs/* docsite/r/.
```

Comment:
Not sure I follow -- this is just copying the static files, what would be the failure mode of concern? (That stated, we obviously haven't tried this workflow yet since it actually deploys the live docsite -- first attempt certainly could get interesting)